packages/tools/client-debugger/client-debugger-view/src/components/AudienceStateTable.tsx

```diff
@@ -47,8 +52,18 @@ export function AudienceStateTable(props: AudienceStateTableProps): React.ReactE
 				<TableRow>
 					{audienceStateColumns.map((column, columnIndex) => (
 						<TableHeaderCell key={columnIndex}>
-							{column.columnKey === "clientId" && <Person24Regular />}
-							{column.columnKey === "userId" && <Person24Regular />}
+							<TooltipHost
+								content="Represents the connection between Fluid Runtime and the Fluid server."
+								id={clientIdTooltipId}
+							>
+								{column.columnKey === "clientId" && <Person24Regular />}
+							</TooltipHost>
+							<TooltipHost
```

Comment:
My thoughts, also interested in others' thoughts on this: I probably wouldn't call out that this is randomly generated. You might just note that the User ID is an application / service concept (optionally) provided to the Framework.

Reply:
I'm ok skipping the "random" part, but probably would keep `Auto-generated by the framework if not specified by the application.` or something along those lines.

Reply:
Changed to `Represents the application specific user identifier provided to the Fluid Framework. Auto-generated by Fluid if not specified by the application.`  😄